lib/index.js

```diff
@@ -186,17 +122,12 @@ function stringify(opts, callback) {
     });
 
     passthroughStream.on('end', function() {
-        // if we didn't already error and exit
-        if (done === false) {
-            return callback(null, stringified);
-        }
-        return null;
+        return callback(null, stringified);
```

Comment:
Do we need to wrap the callback in a strict once?

Reply:
Good idea - added.